File: app/src/main/java/com/galeria/android/AlbumItem.java
package com.galeria.android;

final class AlbumItem {
    final String key;
    final String name;
    final int count;
    final MediaItem cover;
    final long latestDate;

    AlbumItem(String key, String name, int count, MediaItem cover, long latestDate) {
        this.key = key;
        this.name = name;
        this.count = count;
        this.cover = cover;
        this.latestDate = latestDate;
    }
}

File: app/src/main/kotlin/com/galeria/android/AlbumItem.kt
package com.galeria.android

class AlbumItem(
    @JvmField val key: String,
    @JvmField val name: String,
    @JvmField val count: Int,
    @JvmField val cover: MediaItem?,
    @JvmField val latestDate: Long,
    @JvmField val firstDate: Long,
    @JvmField val totalSize: Long,
    path: String?
) {
    @JvmField val path: String = path ?: ""
}
